tests/integration_tests/test_sklearn.py

```diff
@@ -2,15 +2,19 @@
 from sklearn.datasets import make_blobs
 from sklearn.exceptions import NotFittedError
 from sklearn.linear_model import SGDClassifier
+from typing import Optional
 
 from optuna import distributions
 from optuna import integration
 
+import numpy as np
+
 
 @pytest.mark.parametrize('enable_pruning', [True, False])
+@pytest.mark.parametrize('fit_params', [None, 'sample_weight'])
```

Comment:
This is rather my taste, not comment.
However, I would use empty string instead of `None` as it avoids the import of `Optional`.